mindsdb/integrations/handlers/cassandra_handler/cassandra_handler.py

```diff
@@ -1,4 +1,7 @@
 from mindsdb.integrations.handlers.scylla_handler import Handler as ScyllaHandler
+from mindsdb.integrations.libs.response import (
+    HandlerResponse as Response)
+import pandas as pd
```

Comment:
Let's remove this import. Since it is not being used, our CI checks are failing.

Reply:
Sorry i missed it